firebase-functions/ktx/src/androidTest/kotlin/com/google/firebase/functions/ktx/CallTests.kt

```diff
@@ -0,0 +1,101 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.firebase.functions.ktx
+
+import com.google.common.truth.Truth.assertThat
+import com.google.firebase.FirebaseApp
+import com.google.firebase.ktx.Firebase
+import com.google.firebase.ktx.app
+import com.google.android.gms.tasks.Tasks
+import org.junit.BeforeClass
+import org.junit.Test
+import org.junit.runner.RunWith
+import android.support.test.InstrumentationRegistry
+import android.support.test.runner.AndroidJUnit4
+
+@RunWith(AndroidJUnit4::class)
+class CallTests {
+    companion object {
+        lateinit var app: FirebaseApp
+
+        @BeforeClass @JvmStatic fun setup() {
+            FirebaseApp.initializeApp(InstrumentationRegistry.getContext())
```

Comment:
can be changed to `app = Firebase.initialize(InstrumentationRegistry.getContext())` ?

Reply:
Also I think it is a good idea to `app.delete()` in the `@AfterClass` method, otherwise adding more test classes will make tests fail

Reply:
Done. Good catch with the missing app.delete()

Reply:
Also, I needed to force it to fail when null (!!) because without it the compiler complains about getting a FirebaseApp? when expecting FirebaseApp.